pkg/hs/urbit-king/lib/Urbit/King/CLI.hs

```diff
@@ -221,6 +227,17 @@ pillFromURL = PillSourceURL <$> strOption
 pierPath :: Parser FilePath
 pierPath = strArgument (metavar "PIER" <> help "Path to pier")
 
+injectEvents :: Parser [Injection]
+injectEvents = many (InjectOneEvent <$> strOption ( short 'I'
+                          <> long "inject-event"
+                          <> metavar "JAM"
+                          <> help "Path to a jammed event")  <|>
+                     InjectManyEvents <$> strOption (
+                        long "inject-event-list"
+                     <> metavar "JAM LIST"
+                     <> help "Path to a jammed list of events"
```

Comment:
I think you want these options to be `hidden` so they're not listed in the main usage string, only in the extended list of options.

Reply:
Good catch. I keep forgetting about this since that's a weird default behaviour.